### PR TITLE
New prop vehicles

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Ports/edits some vehicle props from CM.

Bigger truck, some new directionals for some existing props, and some snow covered and wrecked variants.

Cleaned up some dup vars in vehicle props.
## Why It's Good For The Game
Mapping props gud.
## Changelog
:cl:
imageadd: added some new vehicle props
/:cl:

### CODE REVIEW

File: code/game/objects/structures/prop.dm [hunk 1]
```diff
@@ -1191,12 +1207,76 @@
...
+	icon = 'icons/obj/vehicles/large_truck.dmi'
+	icon_state = "truck"
+	coverage = 80
+	bound_height = 128
+	bound_width = 128
+	resistance_flags = XENO_DAMAGEABLE
+
+/obj/structure/prop/vehicle/big_truck/Initialize(mapload)
+	. = ..()
+	setDir(dir)
```
Comment (TiviPlus) [inline code/game/objects/structures/prop.dm:1222]: ? doesnt mapload call this

**Review by TiviPlus — COMMENTED** [review]
rest lgtm

File: code/game/objects/structures/prop.dm (hunk 1 above)
Comment (Lumipharon) [reply code/game/objects/structures/prop.dm:1222]: doesn't appear to, at the very least it doesn't if you admin spawn something

**Review by Wirpy — APPROVED** [review]
cant be worse than porting tg sprites am i rite

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**